### DISCUSSION TITLE
Trying to serve HTML and CSS from static folder.

### DISCUSSION THREAD

**Discussion by AltusJVR** [rwf2/Rocket#1797] [Questions]

**Comment by blackghost1987** [comment]
I'm not sure if this is still active, because the description got deleted, but I'll answer it anyway, just because it's still "Unanswered".
You can drop the ".." from "../css/index.css", because the html is already in the static directory, so the relative path is just "css/index.css".
Also you don't have to mount the "static/css" path either, StaticFiles will serve the subdirectories as well, every file in the tree is accessible.

Don't forget to close the question if it's not relevant anymore.

**Reply by AltusJVR** [reply]
Thank you.
